src/demands/demands.model.ts

```diff
@@ -0,0 +1,22 @@
+import { BelongsTo, Column, DataType, Model, Table } from 'sequelize-typescript';
+import { Supply } from '../supplies/supplies.model';
+import { Priority } from '../priorities/priorities.model';
+import { Place } from '../places/places.model';
+
+@Table({ tableName: 'supplies', underscored: true })
+export class Demand extends Model {
+  @Column({ type: DataType.UUID, primaryKey: true, allowNull: false })
+  id!: string;
+
+  @Column({ allowNull: true, type: DataType.STRING })
+  comment!: string | null;
+
+  @BelongsTo(() => Place)
+  place!: Place;
+
+  @BelongsTo(() => Supply)
+  supply!: Supply;
```

Comment:
Consider typing it in TypeScript as Pomise<Supply> or Supply | null. This is because we are not loading entities eagerly but with lazy load. This will prevent from using a field if it is not fetched :)

Reply:
Same about other BelongsTo  and HasMany

Reply:
Good point - I forgot to add union with `null`.